mmv1/third_party/terraform/utils/config.go.erb

```diff
@@ -575,6 +576,16 @@ func (c *Config) NewSqlAdminClient(userAgent string) *sqladmin.Service {
 	return clientSqlAdmin
 }
 
+func (c *Config) NewAlloydbClient() *alloydb.AlloyDBAdminClient {
+	log.Printf("[INFO] Instantiating Alloydb client")
+	clientAlloydb, err := alloydb.NewAlloyDBAdminRESTClient(c.context, option.WithHTTPClient(c.Client))
```

Comment:
Is it possible to support end-point override functionality in alloydb client also, like other clients?

Reply:
yes